content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -212,14 +219,25 @@ When the feature gate is enabled, you can set the `protocol` field of a NetworkP
 You must be using a {{< glossary_tooltip text="CNI" term_id="cni" >}} plugin that supports SCTP protocol NetworkPolicies.
 {{< /note >}}
 
+# What you CAN'T do with network policy's (at least, not yet)
 
+As of Kubernetes 1.20, the following functionality does not exist in the NetworkPolicy API, but you might be able to implement workarounds using Operating System components (such as SELinux, OpenVSwitch, IPTables, and so on) or Layer 7 technologies (Ingress controllers, Service Mesh implementations) or admission controllers.  In case you are new to network security in Kubrnetes, its worth noting that the following User Stories cannot (yet) be implemented using the NetworkPolicy API.  Some (but not all) of these user stories are actively being discussed for future releases of the NetworkPolicy API.
 
+- Forcing internal cluster traffic to go through a common gateway (this might be best served with a service mesh or other proxy).
+- Anything TLS related (use a service mesh or ingress controller for this).
+- Node specific policies (you can use CIDR notation for these, but you cannot target nodes by their Kubernetes identities specifically).
+- Targeting of namespaces or services by name (you can, however, target pods or namespaces by their{{< glossary_tooltip text="labels" term_id="label" >}}, which is often an viable workaround).
+- Creation or management of "Policy requests" that are fulfilled by a third party.
+- Default policies which are applied to all namespaces or pods (there are some third party Kubernetes distributions and projects which can do this).
+- Advanced policy querying and reachability tooling.
+- The ability to target ranges of Ports in a single policy declaration.
+- The ability to log network security events (for example connections that are blocked or accepted).
+- The ability to explicitly deny policies (currently the model for NetworkPolicies are deny by default, with only the ability to add allow rules).
```

Comment:
I've a specific question on this (sorry, didn't tested yet): when we define a ingress policy, it's assumed that all the ingress traffic will be blocked BUT that specified? Is this per namespace or per Pod selector? 

As far as I remember, the behavior from ingress policy is different from egress policy (I remember a discussion about this with Casey or Chris Luciano), so maybe in this or in a later commit this could be explored (and maybe drawed as a flow, don't know) so that users would know that ingress and egress have different default behavior :)

Reply:
I thought egress deny all is the same except usually people open up port 53 so that at least dns can work, so that later once real egress allows are layered on the deny the policies work intuitively .

Lmk if u know if anything different I'm happy to test for you.

Reply:
Yeap, the 'default deny' example. My point here was just that as far as I remember the behavior of ingress and egress are different when a first 'specific' rule was created, but in this case nevermind 

I've left some other reviews above